src/libsystemd/sd-bus/bus-introspect.c

```diff
@@ -101,6 +101,11 @@ static int introspect_write_arguments(struct introspect *i, const char *signatur
 
                 fprintf(i->f, "   <arg type=\"%.*s\"", (int) l, signature);
 
+                if (**names) {
```

Comment:
hmm, according to CODING_STYLE we say that C's downgrade-to-bool feature should only be used for actual bools or bool-like values, as well as for ptrs, but not for numeric values or characters. Hence,please if write this as `if (**names != 0)` or even `if (**names != '\0')`...